test/test.go

```diff
@@ -22,9 +22,10 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/kubernetes/scheme"
 
+	"github.com/kubermatic/utils/pkg/testutil"
```

Comment:
3 groups, not 4; check your IDE settings

Reply:
fixed